pkg/limayaml/limayaml.go

```diff
@@ -31,6 +31,7 @@ type LimaYAML struct {
 	UseHostResolver   *bool             `yaml:"useHostResolver,omitempty" json:"useHostResolver,omitempty"` // DEPRECATED, use `HostResolver.Enabled` instead
 	PropagateProxyEnv *bool             `yaml:"propagateProxyEnv,omitempty" json:"propagateProxyEnv,omitempty"`
 	CACertificates    CACertificates    `yaml:"caCerts,omitempty" json:"caCerts,omitempty"`
+	ExtraArchive      *File             `yaml:"extraArchive,omitempty" json:"extraArchive,omitempty"`
```

Comment:
```suggestion
	ExtraArchives      []File             `yaml:"extraArchives,omitempty" json:"extraArchives,omitempty"`
```
for supporting multi-arch